.github/workflows/trivy.yml

```
@@ -12,11 +12,16 @@ jobs:
   generate-sbom:
     runs-on: ubuntu-latest
     steps:
+      - name: Harden the runner (Audit all outbound calls)
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)